Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class Lesson3
+    {
+        public static void Demo()
+        {
+            string firstName = InputString("Enter First Name: ");
+
+            string lastName = InputString("Enter Last Name: ");
+
+            int personsAge = InputInt("Enter Age: ");
+
+            float personsWeight = InputFloat("Enter Weight in kg: ");
+
+            float personsHeight = InputFloat("Enter Height in m:");
+
+            Console.WriteLine(firstName + " " + lastName + " is " + personsAge + " years old, their weight is "
+                                + personsWeight + " and their height is " + personsHeight + " m.");
+
+            Console.WriteLine("Their BMI is " + string.Format("{0:0.#}", CalculateBMI(personsWeight, personsHeight)));
+
+            Console.WriteLine();
+            Console.WriteLine("Please Enter a second persons information");
```

Comment:
instead of copy-pasting code from second person- reuse it via a function. Put the way you work with first person to a function and only change the person prompt (which you will have to pass from an input).

Reply:
Yes - missed that makes sense to have that as a function.